Automattic/simplenote-ios
# Fix editor font heirarchy

### Fix
Modify the size and weight of the note **title** in the editor view

Text size normal:

Before             |  After
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/36532468/75713553-f067cf00-5cc1-11ea-944c-0403473f9a74.PNG)  |  ![](https://user-images.githubusercontent.com/36532468/75800563-ab9b7100-5d71-11ea-9e17-da2734299dc2.PNG)

### Test
1. Press on any note
2. Notice that the note title is larger
3. Test that text sizes dynamically

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
